docs/guide-first.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Run Your First Enclave"
+layout: docs-enclaver-single
+category: guides
+weight: 1
+---
+
+# Build and Run Your First Enclave
+
+This guide will walk you through booting an EC2 machine, configuring it, then building and running your first enclave with Enclaver.
+
+## Boot the Machine
+
+Only certain [EC2 instance types][instance-req] can run Nitro Enclaves. Since dedicated CPUs are carved off for the enclave, the larger x86 types with 4+ vCPUs are ok, with `c6a.xlarge` being the cheapest. Your machines must be booted with the Nitro Enclave option enabled, which is found under the "Advanced details" > "Nitro Enclave". 
+
+See [the Deploying on AWS](deploy-aws.md) for more details and an example CloudFormation template.
+
+## Configure Instance
+
+First, SSH to your EC2 machine:
+
+```sh
+$ ssh ec2-user@<ip address>
+```
+
+Install the Nitro Enclave packages:
+
+```
+$ sudo amazon-linux-extras install aws-nitro-enclaves-cli -y
```

Comment:
`sudo yum install -y amazon-linux-extras aws-nitro-enclaves-cli` - right?

Reply:
yeah it worked either way